Source/FakeItEasy/Creation/DefaultFakeCreatorFacade.cs

```diff
@@ -126,6 +126,11 @@ public IFakeOptionsBuilder<T> Implements(Type interfaceType)
                 return this;
             }
 
+            public IFakeOptionsBuilder<T> Implements<TInterface>()
+            {
+                return Implements(typeof (TInterface));

```

Comment:
@adamralph, thanks for commenting. That matched my understanding—that contributors don't need to do anything to see the warnings.

And yeah, I hope I didn't give the impression that accepting the PR was contingent on @thomaslevesque fixing the warnings. I was really after the tests, and figured that were I the contributor, I'd've preferred the project owner to ask me for the warnings to be fixed than to learn later that they'd had to go in and fix them.